docs/migrations/web-twig/MIGRATION-v3.md

```diff
@@ -163,6 +164,14 @@ remains accessible via the `isDockedOnMobile` property.
 
 Add `isDockedOnMobile` prop to the `ModalDialog` component.
 
+### Tabs: TabLink `target` Prop
+
+The `target` prop was renamed to `targetPaneId` in the `TabLink` component.
```

Comment:
Plus the reasoning maybe? I like how @curdaj added the (apparently MDN 🙂) guidelines for the `role` attribute.